tracer.go

```diff
@@ -25,6 +25,10 @@ type Options struct {
 	TrimUnsampledSpans bool
 	// Recorder receives Spans which have been finished.
 	Recorder SpanRecorder
+	// An EventManager can be used to enhance the tracer by effectively
+	// attaching external code to trace events. See NetTraceIntegrator
```

Comment:
should also point the reader to the `Event` type for a schema of possible param values.